### PR TITLE
Defining the zip attributes based on the xsd schema

### PR DESCRIPTION
**Pull request by eduard13**
## Purpose of this pull request

This pull request (PR) provides more info regarding the zip attributes which are allowed on defining the allowed zip code patterns for some countries. It also provides a small example how to pass multiple zip patterns for the same country. 

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.1/howdoi/checkout/checkout_zip.html
- https://devdocs.magento.com/guides/v2.2/howdoi/checkout/checkout_zip.html
- https://devdocs.magento.com/guides/v2.3/howdoi/checkout/checkout_zip.html

## Links to Magento source code

- https://github.com/magento/magento2/blob/2.1/app/code/Magento/Directory/etc/zip_codes.xsd

whatsnew
Added reference documentation for [ZIP code attributes](https://devdocs.magento.com/guides/v2.3/howdoi/checkout/checkout_zip.html), including a new code sample showing how to pas multiple ZIP code patterns for the same country

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.1/howdoi/checkout/checkout_zip.md [hunk 1]
```diff
@@ -17,6 +17,28 @@ When a shopper specifies the country and ZIP code in the shipping address during
 In Magento the input masks for the **ZIP code** field are specified in the `<Magento_Directory_module_dir>/etc/zip_codes.xml`. Input masks are specified per country, and are entered in the form of regular expressions.
 The syntax of defined by the [zip_code.xsd]({{ site.mage2bloburl }}/{{ page.guide_version }}/app/code/Magento/Directory/etc/zip_codes.xsd) scheme.
 
+The following table defines the `zip` node attributes:
+
+ Attribute name | Required | Description
+--- | --- | ---
+`countryCode` | Yes | The country code (Alpha-2 format) for which the zip is defined
+
```
Comment (rogyar) [inline guides/v2.1/howdoi/checkout/checkout_zip.md]: Please, provide also an example of the `zip` node. So in result we have an example for the `zip` node and an example for the `codes` node.
Thank you

Comment (eduard13) [reply guides/v2.1/howdoi/checkout/checkout_zip.md]: Hi @rogyar a `zip` node example has been provided.
Thank you.

**Comment by jeff-matthews** [comment]
running tests

**Pull request merged**